packages/angular/projects/clr-angular/src/forms/combobox/_combobox.clarity.scss

```diff
@@ -35,26 +49,6 @@
       $clr-use-custom-properties
     );
 
-    border-bottom: $clr-global-borderwidth solid $clr-forms-border-color;
-
-    // the border color is on the component; we don't want it on the input itself
-    .clr-input.clr-combobox-input:focus {
-      background: none;
-    }
-
-    // ideally I want to just use &:focus-within, but cross-browser support is limited
-    // for now, we programmatically add/remove .clr-focus class to achieve this
-    &:focus-within,
-    &.clr-focus {
-      border-bottom: $clr_baselineRem_2px solid $clr-forms-focused-color;
-    }
-
-    &.invalid,
-    &.invalid:focus,
-    &.clr-focus.invalid {
-      border-bottom-color: $clr-forms-invalid-color;
-    }
```

Comment:
Is this logic (underline it red when there's error) being handled in the new code? Or no longer needed?